db/migrate/20170326034952_create_students.rb

```diff
@@ -2,6 +2,7 @@ class CreateStudents < ActiveRecord::Migration[5.2]
   def change
     create_table :students do |t|
       t.integer :cpf
+      t.string :name
```

Comment:
This add is ok, but if you enter this [db/seed/development.rb](https://github.com/jamessom/scms/blob/fix/model_name/db/seeds/development.rb) will not work correctly.